src/WebJobs.Script/Extensions/ScriptLoggingBuilderExtensions.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Extensions.Logging
+{
+    public static class ScriptLoggingBuilderExtensions
+    {
+        internal static readonly string[] AllowedCategoryPrefixes = new[]
+        {
+            "Microsoft.Azure.WebJobs",
+            "Function",
+            "Worker",
+            "Host"
+        };
```

Comment:
I went ahead and added an "allowed category" list for now to filter everything out but our logs. We can build up from here. 

It's overrideable in host.json if someone wants to increase their log scope for their own logs, but that will not affect the System logs. They remain going through the filter.